stacks/spica/src/bucket/pages/index/index.component.ts

```diff
@@ -186,6 +194,9 @@ export class IndexComponent implements OnInit, OnDestroy {
       }),
       tap(schema => {
         Object.keys(schema.properties).map(key => {
+          schema.properties[key]["icon"] = this.systemFields.find(
```

Comment:
Instead of manipulating the original schema.properties, we should manipulate the deep copy object. Also, instead of using schema.properties in view, we should use a deep copy object there as well. As I see, using `displayedProperties` will be a better implementation.